Lib/test/test_traceback.py

```diff
@@ -927,7 +927,17 @@ def f():
         ]
         self.assertEqual(actual, expected)
 
+    def test_memory_error(self):
+        def f():
+            raise MemoryError()
 
+        actual = self.get_exception(f)
+        expected = ['Traceback (most recent call last):',
+            f'  File \"{__file__}\", line {self.callable_line}, in get_exception',
```

Comment:
Why are double quotes escaped?

Reply:
Oh that's because I started with double quotes for these strings and changed it later to single quotes but didn't remove the scapes